addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
Yes. This fixes the immediate problem. I think we can also improve by not having spaniel schedule any work in certain cases. I believe the memory leak is caused by work being scheduled but never popped off the stack. This will ensure the work is popped so the stack doesn't grow uncontrollably. `memwatch` would be awesome, I would definitely approve that PR :)